tests/spec/test-run-api-service.js

```diff
@@ -406,6 +406,29 @@
                     fail.should.not.have.been.called;
                 });
             });
+
+            it('should call GET with & in variable name', function () {
+                var rs = new RunService({ account: account, project: project });
+                rs.query({ '&saved': true, '.price&': '1' });
+                var req = server.requests.pop();
+                req.url.should.equal(baseURL + ';&saved=true;.price&=1/');
+            });
+
+            it('should call GET with & in long variable names', function () {
```

Comment:
I'd remove this test from here and add an equivalent version in test-variables-api-service.js. That should already have utility methods to do long GETS